privacy_policy/en/privacy-main.md

```diff
@@ -0,0 +1,49 @@
+# Adguard Privacy Policy
+*August 03, 2017*
+
+> **Keynote:** Adguard does not share/sell any of your personal data. We are proud to say that we are one of the fighters for users privacy and we are strongly committed to this principle and to being as transparent as possible.
+
+Our privacy policy lists the information that is collected and the processing applied to it. All the info collected includes no more than is crucial to provide full functionality of Adguard products, and is never shared with any third parties. We do not collect anything for tracking purposes and take all necessary technical, administrative and physical measures to protect the information we get.
+
+Please read our privacy policy to learn what kind of information we collect via Adguard products and how we store and use it afterwards.
+
+## What we consider as “personal information”
+
+Personal information or Personally identifiable information is information that can be used on its own or with other information to identify, contact, or locate a single person, or to identify an individual in context. 
+
+Any other information (that does not identify you in any way) is considered as non-personal.
+
+Our general principle is to collect only the information that is necessary to provide you with full functionality of our products and for licensing. And we never share any of your data with third-parties.
+
+## How do we get information about you?
+
+There are several ways how we may get your data:
+
+* When you give us the information directly (when you yourself contact us or send us bug report, or subscribe to our blog, forum, newsletter and etc).
+* When you are using our products, information is automatically collected. It occurs when you are updating programs, contact support services and so on. You can find the full list of data collected on a Privacy Notice page dedicated to the product you use.
+* When you visit our website, there are cookies and metrics that give us information about you. Learn more about the data collected and how you can manage that on the [Adguard websites’ privacy notice](the new link will be here). 
```

Comment:
You can come up with the links already.

For instance:
https://adguard.com/privacy-websites.html